src/img/index.js

```diff
@@ -12,6 +12,10 @@ import jackolantern from './items/jackolantern.png'
 import rainbowCheese from './dishes/rainbow-cheese.png'
 import spaghetti from './dishes/spaghetti.png'
 
+import asparagus from './items/asparagus.png'
+import asparagusGrowing from './items/asparagus-growing.png'
+import asparagusSeed from './items/asparagus-seed.png'
+import asparagusSprout from './items/asparagus-sprout.png'
```

Comment:
Let's remove this to keep it out of the app bundle.

Reply:
good eye! removed the sprout refs from this file